packages/RatPack.package/RPSeparator.class/instance/nameRoute..st

```diff
@@ -0,0 +1,25 @@
+as yet unclassified
+nameRoute: aClassName
+
+	| char separatorPositions nextChar secondNextChar routeName lastSeparatorPosition |
+
+	separatorPositions := OrderedCollection new.
+	1 to: aClassName size do: [ :index | 
+		char := aClassName at: index.
+		(index + 2 <= aClassName size ) ifTrue: [
+			nextChar := (aClassName at: (index + 1)) asCharacter.
+			secondNextChar := (aClassName at: (index + 2)) asCharacter.
+			((nextChar isUppercase and: [ secondNextChar isLowercase. ]) or: 
+			(nextChar isUppercase and: [ secondNextChar isUppercase and: [char isLowercase ]] )) ifTrue: [
+				separatorPositions add: index. ]. ]. ].
```

Comment:
Could we put finding the seperator positions in an own method? 25 lines is pretty long